### ISSUE TITLE
Cant pick already uploaded video

### ISSUE THREAD

**Issue by lotarbo** [awcodes/filament-curator#177]
Hello, i have field
```
CuratorPicker::make('video')
                ->acceptedFileTypes(['video/*'])
                ->constrained()
                ->required(),
```
i can add new video, but i cant pick existing video

**Comment by awcodes** [comment]
Can you provide a reproduction repo? I can't replicate this or I'm not understanding the issue.

**Comment by lotarbo** [comment]
@awcodes i make video, hope it can help. If no, i will make repo)


https://github.com/awcodes/filament-curator/assets/1766890/31a60142-046d-4f71-909e-1848a13e8216

**Comment by awcodes** [comment]
Ok, this is happening because your accepted file types is too generic. The thumbnails are looking for actual mime types for the video. I.e. when the video is saved it's mime type isn't 'video/*' it's propbably 'video/mp4' so it's getting disabled as a selectable option.

**Comment by lotarbo** [comment]
@awcodes so, its not a bug and i must use strict mime type?

**Comment by awcodes** [comment]
I'll make a note to look into a way to maybe regex it. But for now you would need to use strict mime types.

Or you're welcome to PR the fix. 😄

**Comment by awcodes** [comment]
Resolved with v2.8.0
